CHANGELOG-3.2.md

```diff
@@ -1,5 +1,22 @@
 
 
+## [v3.2.16](https://github.com/coreos/etcd/releases/tag/v3.2.16) (2018-02-TBD)
+
+See [code changes](https://github.com/coreos/etcd/compare/v3.2.15...v3.2.16) and [v3.2 upgrade guide](https://github.com/coreos/etcd/blob/master/Documentation/upgrades/upgrade_3_2.md) for any breaking changes.
+
+### Fixed(v3)
+
+- Fix [`mvcc` "unsynced" watcher restore operation](https://github.com/coreos/etcd/pull/9281).
```

Comment:
Do we already have this backported to 3.2? If not, we can use this PR: https://github.com/coreos/etcd/pull/9297

Reply:
Just updated. Thanks.

Will push before actual release.